MAUI/DataForm/getting-started.md

```diff
@@ -0,0 +1,491 @@
+
+{% tabs %}
+{% highlight MainPage.xaml.cs %}
+
+    public class ViewModel
+    {
+        public DataModel DataModel {get; set;}
+        
+        public ViewModel()
+        {
+            this.dataModel = new DataModel();
+        }
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+## Setting data object
+
+To populate the labels and editors in the data form, set the `DataObject` property.
+
+{% tabs %}
+{% highlight MainPage.xaml %}
+
+    <?xml version="1.0" encoding="utf-8" ?>
+    <ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+                xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+                xmlns:dataForm="clr-namespace:Syncfusion.Maui.DataForm;assembly=Syncfusion.Maui.DataForm"
+                x:Class="GettingStarted.MainPage">
+
+                    <ContentPage.BindingContext>
+                        <local:ViewModel/>
+                    </ContentPage.BindingContext>    
+	                        <dataForm:SfDataForm x:Name="dataForm" 
+						    DataObject="{Binding DataModel}"/>    
+    </ContentPage>
+
+{% endhighlight %}
+{% highlight MainPage.xaml.cs %}
+
+    dataForm.DataObject = new DataModel();
+
+{% endhighlight %}
+{% endtabs %}
+
+Now, run the application to render the `data form` to edit the data object as in the following screenshot:
+
+`Image`
+
+## Defining editors
+
+The data form control automatically generates `DataFormItems` (which has UI settings of data field) when the data object set to the `SfDataForm.DataObject` property. The `DataFormItem` encapsulates the layout and editor setting for the data field appearing in the data form. When the `DataFormItems` are generated, you can handle the SfDataForm.GenerateDataFormItem event to customize or cancel the `DataFormItem`. 
+
+The type of input editor generated for the data field depends on the type and attribute settings of the property. The following table lists the `DataFormItem` and its constraints for generation:
+
+<table>
+<tr>
+<th>Generated DataFormItem Type</th>
+<th>Data Type / Attribute</th>
+</tr>
+<tr>
+<td>
+{{'[DataFormTextItem]
+`Link`
+</td>
+<td>
+Default DataFormItem generated for the String type and the properties with [DataType(DataType.Text)], [DataType(DataType.MultilineText)] and [DataType(DataType.Password)] attributes.
+</td>
+</tr>
+<tr>
+<td>
+{{'[DataFormCheckBoxItem]`Link`
+</td>
+<td>
+Generated for the Bool type property.
+[BoolDataTypeAttribute]
+</td>
+</tr>
+<tr>
+<td>
+{{'[DataFormSwitchItem] `Link`
+</td>
+<td>
+Generated for the Bool type property.
+[BoolDataTypeAttribute]
+</td>
+</tr>
+
+<tr>
+<td>
+{{'[DataFormDateItem] `Link`
+</td>
+<td>
+Generated for the DateTime type property.
+[DataType(DataType.Date)].
+[DataType(DataType.DateTime)].
+</td>
+</tr>
+<tr>
+<td>
+{{'[DataFormTimeItem]
+`Link`
+</td>
+<td>
+Generated for the DataTime type property.
```

Comment:
Generated for the TimeSpan type property. 

Also include => TimeOnly

Reply:
TimeSpan, TimeOnly added